quickfixj-core/src/main/doc/usermanual/usage/configuration.html

```diff
@@ -1285,11 +1285,21 @@ <H3>QuickFIX Settings</H3>
   <TR ALIGN="left" VALIGN="middle">
     <TD><I>TestRequestDelayMultiplier</I></TD>
     <TD>Fraction of the heartbeat interval which defines the additional time to wait
-    if a TestRequest sent after a missing heartbeat times out.
+    if a TestRequest sent after a missing heartbeat times out (final coefficient value is equal to
+    TestRequestDelayMultiplier + 1.0).
     </TD>
-    <TD>0..1</TD>
+    <TD>any non-negative value</TD>
     <TD>0.5</TD>
   </TR>
+  <TR ALIGN="left" VALIGN="middle">
+      <TD><I>HeartBeatTimeoutMultiplier</I></TD>
+      <TD>Fraction of the heartbeat interval which defines the additional time to wait
+      since the last message was received before disconnecting (final coefficient value is equal to
+      HeartBeatTimeoutMultiplier + 1.0).
+      </TD>
+      <TD>any non-negative value</TD>
+      <TD>1.4</TD>
```

Comment:
The default hardcoded values was 2.4, but +1 is always done in the code the same as for TestRequestDelayMultiplier